apps/estuary/mix.exs

```diff
@@ -28,13 +28,15 @@ defmodule Estuary.MixProject do
 
   defp deps do
     [
-      {:credo, "~> 1.1", only: :dev, runtime: false},
+      {:credo, "~> 1.1", only: [:dev, :test], runtime: false},
       {:dead_letter, in_umbrella: true},
       {:distillery, "~> 2.1"},
       {:divo, "~> 1.1", only: [:dev, :integration]},
       {:divo_kafka, "~> 0.1.5", only: [:dev, :integration]},
       {:elsa, "~> 0.10.0"},
-      {:mock, "~> 0.3", only: [:test, :integration], runtime: false},
+      {:jason, "~> 1.1"},
```

Comment:
where is jason being used? if it's just in tests can that be scoped to just those tests or does that cause warnings about other umbrella apps that use it in their prod code?

Reply:
We're using it in the data_writer_test and message_handler_test. However, I removed it from the mix and did a mix deps.get and they're still passing, so I'll remove it from the mix.

Reply:
You’d still have it unless you removed it from the lock file and cleaned the deps with `mix deps.unlock —unused` and `mix deps.clean —unused`